data/sql/updates/pending_db_world/rev_1595505810975581600.sql

```diff
@@ -0,0 +1,33 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1595505810975581600');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 25773;
+
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 25773) AND (`source_type` = 0) AND (`id` IN (0));
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(25773, 0, 0, 0, 54, 0, 100, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Fizzcrank Survivor - On Just Summoned - Say Line 0');
+
+UPDATE `creature_template` SET `unit_flags` = 2 WHERE (`entry` = 25773);
```

Comment:
```suggestion
UPDATE `creature_template` SET `unit_flags`=`unit_flags`|2, `AIName` = 'SmartAI' WHERE `entry` = 25773;

DELETE FROM `smart_scripts` WHERE (`entryorguid` = 25773) AND (`source_type` = 0) AND (`id` IN (0));
INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
(25773, 0, 0, 0, 54, 0, 100, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Fizzcrank Survivor - On Just Summoned - Say Line 0');
```

Reply:
no need to update the same creature two times.
also please use ``` `unit_flags`=`unit_flags`|2 ``` to add a flag and not override

Reply:
Yes, it turns out to be optional.